pkg/console/operator/operator.go

```diff
@@ -148,15 +162,22 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 		}
 	}
 
-	if err := c.handleSync(operatorConfig, consoleConfig); err != nil {
+	// we need infrastructure config for apiServerURL.
+	// it is helpful but not essential so we should not die if its not available.
```

Comment:
Is there ever a time we'd expect this not to work? I feel like we should just fail here. Public URL is needed for a few things in console.

Reply:
Well, I figure we need it only for a few small things, that we can guard in console code. So console can be 99% useful without it. Once it's available, the new pods will roll out anyway.

Reply:
It should always be available right? The drawback is that we're not reporting an error anywhere (except in the operator log). You won't know something is broken.

Reply:
Ill update.